### PR TITLE
Update set-the-caller-id-for-a-user.md

### PR DESCRIPTION
**Pull request by pupara**
Added Notes
Replaced SkypeOnlineConnector with MicrosoftTeams

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/set-the-caller-id-for-a-user.md [hunk 1]
```diff
@@ -75,11 +75,12 @@ To learn more about these settings and how you can use them, go [How can caller
     
 2. In the **Windows PowerShell** window, connect to your Microsoft 365 or Office 365 by running:
     
+
    > [!NOTE]
-   > You only have to run the **Import-Module** command the first time you use the Skype for Business Online Windows PowerShell module.
-   > 
+> Skype for Business Online Connector is currently part of latest Teams PowerShell Module.
```
Comment (JohanFreelancer9) [inline Teams/set-the-caller-id-for-a-user.md:80]:
```suggestion
> The Skype for Business Online Connector is currently part of the latest Teams PowerShell module.
```

File: Teams/set-the-caller-id-for-a-user.md [hunk 2]
```diff
@@ -75,11 +75,12 @@ To learn more about these settings and how you can use them, go [How can caller
     
 2. In the **Windows PowerShell** window, connect to your Microsoft 365 or Office 365 by running:
     
+
    > [!NOTE]
```
Comment (JohanFreelancer9) [inline Teams/set-the-caller-id-for-a-user.md:79]:
```suggestion
  > [!NOTE]
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@LanaChin PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**